app/controllers/evidence_hub_controller.rb

```diff
@@ -1,18 +1,44 @@
 class EvidenceHubController < ApplicationController
   DOCUMENT_TYPES = ['Insight'].freeze
 
-  def index
+  def index    
+    @search_form = EvidenceSummarySearchForm.new(search_form_params)
+
     documents = Mas::Cms::Document.all(
-      params: search_params
+      params: parse_filter_params
     )
     @evidence_summaries = EvidenceSummary.map(documents)
   end
 
   private
 
-  def search_params
-    { document_type: DOCUMENT_TYPES }.tap do |hash|
-      hash[:keyword] = params.require(:keyword) if params[:keyword].present?
-    end
+  def search_form_params
+    { 
+      document_type: DOCUMENT_TYPES 
+    }.merge(evidence_summary_search_form_params)
+
+  end
+
+  def evidence_summary_search_form_params
+    params.fetch(:evidence_summary_search_form, {}).permit(
+      :keyword,
+      :years_of_publication,
+      client_groups: [],
+      topics: [],
+      countries_of_delivery: []
+    )
+  end
+
+  def filter_params
+    evidence_summary_search_form_params.reject{|k,_| !filter_keys.include?(k) }
```

Comment:
rather than using a double negative (`reject` and `!`) would it be more readable to use `select` ?